mappings/a4cdfddb82640194871681aece726d85fd302eb2204c47055d375acb4c415a59.json

```diff
@@ -0,0 +1 @@
4SuhYWwVwwfkz+fw/HcL8hSXsSEwcD+PpjXsohK6ESGLKFWw9AmHcvNRaRNQFO7bAU5vqdO0WjIQQSUs+Y2ksKXIFzdR2xbbekLXRBznvta8ClXUAkB5Yg7Uiab9o0VYy3N/LQM9uBnZsYbBnOyM9W6G6nqLoYlpbyPz5go6D28nPWQDNCqO3Y+OkgrdcQ9Rqe8gxC5gYVGChLcP9jwo+8g+93HVvFSGwQqAT/aEHeDcuJzbVH/4qbti/JihgTLE1wKtxUb1ZUqBxAUHZ1hbwyY8fzHveNYtc4SnoH0Zr6ex4cMMMK6FLVRKu1ZgVCiMsqphHeBZGajCgqXXXGNgZ09UZ0bXb0DcAIzXBcBXwCggZoKV7LI1J9l0qAmlQwhBkJFlqZNQILQ2WSS3Q3gGtM5vJTZsMk6dAoQUrA2ITIaqrkYQIPOd1Gqwk+orrkaFjXJlgo6Sr0/KvV9W48tvDxNqSzJiR1kUA7gTeCzzOX0Fc7E1/bVCk35nmDc4AvgS8EcDzhI5jKwFx+KEFPvbhEue/toTIAwOG2IBEI4bK7NHU3lhsNgMl5/Sx0uUyKhO79i16ikubLg9BNcRUqkSVGnElQkc1jFagQQiJ9A3KM0hf4gUCLx9A3oOMAk9A0IhpOxzrZcFECK0RViDMGmxcRigfqzVmaASpDVjhAl8e0CCpDhl+eG2Ny79dZdO2GMAmDXwVTme4HCcyUgvueXFdPxv6W4AipfE3fD7weWCBECClKx8AOPG4HB+6pJHXnFnAKwrYMUxc1kjluvRbaxFKYptKTh8YlbgaKwzGn4nwpmOE09mENUirwQhEfQOYXiBwhUxKuFEIIyEIgZEG1yzRKT9CWOykwxB+K9LUcS5IHxtvRehOpxQLDQNVTNKTXGWAgqTWb/j57+p8/fsVHn3C3ban0NqgEh32AeAyXCXXX01/mIj+lqBIvz+9+Tbgk8B7gJyUwoI1iQePo4/K8q6Lcpx7iqK5Q0FooWKJDKjmPDLwxwpSEVg00IAIFpBIckYz3YWPjbsh3jo2EFe4ASwMDUM0rl9E4iiwug7NixGluWDqyR8DrO7BxmuRNoMBTLWGV61BXkIA3dsN1/y2xlX/XWPVWgeGRFSkYOjB6VhX4B6Sv6r+MBH9rUGR0niucTguWeT1AGkbSmOQgJjRITnvzCxvODvLMUsVYkoBhA9ljY5xzSKlQKAQmQVYmUPYMUC4wXg1RH0tiJhUzTFY5FDVcYo9ciEE6Aibn4RtXYY0rimHRWFNGVtdjxXaNV73NNQrmGHDvY/F/PxXNa77fZ1du13egJJoC17iaAtxXt/LGWuQ/1fXHyaiFwsoYE9dA1xl04dxCmla5xFr7WZgegqOPtznNa9u4GUvK3DoEh/VkMxFrYXYaDZGTMKI2I1mEiTpdxYbPYG0VdzIaFe8qWtVxEh9tPdmuiRra2hZRLQejVHKpccJL+n49zgENfA8zEDEow8N8ceba/z6TyEProxI+rcapTDWjoIhxsWGvoZzWcOLgDuMpxcTKFIab6EAnITTxM8BskkbQROPOXZE4MHSgzOcfGKeE08IOPLwKcyYuRiZt4y2d4pil99Y24at78TN7Eyeem1gqAJmzLsprEDI2G3Q1MMh3wbK4ICksUMb2b69i4dWae66q8Kf76ywanVImHRyBrSnkNqMjggfAX4BfBPXVQZeZGBI6cUIipT2BsdBODfv64EFApLBbJhYjyb6CECUSj5LFpVYtrSRpQc3snBRkZkzSkxtHaEhtxWVC5zAUsIpmEOV0cE2o1nDGog0JpjJkJjFrq4qW7dXWf/kMI890cNjq7p48smYweFRXdAAxlNIY5Hjeq+uwXGGn+K63U50by8qejGDIqW9N7CIi6Wciyuxn5zGOdwQXKuTllSjIAHIBpKGRsG0KYq2SR5NjZKmRkVWaXyjkZ4LqsUx1ELLwLBmYEjSM5xl5846AwMhtfroGaZpiEZKhJRCpfPDEq7QhWs3eR0uvyQNcadd6V6UYEjpfwIoUpLJa7zdPhl4KfAynJhZAGn/1XQ2B2httR03BICx+97f/du9/m2FQCglJBZh7B4gMDhl8U4cCG5jz3FMae7kixoMKf1PAkVKqUK69xOXBZYCxwDLk38vAEow5mke/9PN65hAnKfplcnPsWKh0XeM4GairML5F+4DVgPVcVdJYxcvOp3hmeh/IijGUwoQeLop5wPtwEJgNjAf50FtB1pxYCnh5q96jDoxiHEFusPJqxc3c2sbjhtsBtbichr2Hvj6PxYI4+l/OijGUyoWUi6yP3vfS15ZHCgCxnwEIe6Jr+MOfX9uZsWeOZL/Y4Ewnv43gWJvGq83pNwkFTnP5vDGg429Pv+/AgR70/9mUOyPJrrvVHzsTf8rD35/9P8BFQE7tmGQjCEAAAAASUVORK5CYII=", "signatures": [{"signature": "07c70665ce42a7d4962855004d73890a8f22115d499e55653296771b5c7c65176099734286056fa414f158e2a4c600d6262ddb3bd4ad54cc6ef9b2db37f47607", "publicKey": "40c3855ab3a0a01e5b704c190204ad76db93fec32513da45c447ef02a0766983"}]}, "description": {"sequenceNumber": 0, "value": "Because sometimes we all are lazy", "signatures": [{"signature": "7ceea7e2a9323373da11c50aff2946202f107677cb58f12c79ac978a28f76de82366b83e4e4a6cffc20be9afa7151ca3e9f571fc778d59bbe559d9130df6320e", "publicKey": "40c3855ab3a0a01e5b704c190204ad76db93fec32513da45c447ef02a0766983"}]}}
```

Comment:
Hi @CardanoTokenIO,

The Twitter account refers to 1f74393e01708131fd3e5563baa62baa967c8b0d770d21e4825f2783 as the policyID for the LAZY token. This one appears to be incorrect.

If the client does not update the Twitter account with the correct policyID within the next 2 days, we will have to close the PR.

Thanks!